### DISCUSSION TITLE
Recommended environment for contributing to DBeaver open-source project

### DISCUSSION THREAD

**Discussion by rjspann9** [dbeaver/dbeaver#35255] [General]
I started making a contribution to version 24.1.2 by forking the dbeaver/debeaver repo to my account.  I had considerable problems getting Eclipse to give me a clean compile, but did finally manage.  Next, when version 24.2.0 came out, I updated my repo to that version, but Eclipse is starting over (it seems) with the compile errors (before I even try to add my contributed code).  My guess is that I am doing something wrong with the environment configuration.  Can someone point me to some documentation of the environment configuration?  Presently, I am using https://dbeaver.com/docs/dbeaver/Develop-in-Eclipse/ which popped up in a Google search, but may be outdated.
Thanks in advance!

**Comment by E1izabeth** [comment]
This wiki page is not outdated.
Maybe you can try using IDEA https://dbeaver.com/docs/dbeaver/Develop-in-IDEA/
